lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -363,6 +372,31 @@ namespace Js
         return library->GetUndefined();
     }
 
+    Var JsBuiltInEngineInterfaceExtensionObject::EntryJsBuiltIn_Internal_ToLengthFunction(RecyclableObject * function, CallInfo callInfo, ...)
+    {
+        EngineInterfaceObject_CommonFunctionProlog(function, callInfo);
+        AssertOrFailFast(args.Info.Count >= 2);
```

Comment:
Assert that its exactly two instead of greater or equal? I have been doing this as an easy way to tell exactly when the JS starts making calls that the platform may not know how to deal with.